sqlalchemy_redshift/dialect.py

```diff
@@ -896,6 +897,15 @@ def create_connect_args(self, *args, **kwargs):
         default_args.update(cparams)
         return cargs, default_args
 
+    @classmethod
+    def dbapi(cls):
+        try:
+            return __import__(cls.driver)
```

Comment:
It looks like https://docs.python.org/3/library/importlib.html#importlib.import_module is preferred over `__import__`.

Reply:
addressed in edfb456